### DISCUSSION TITLE
Is WSLg going to be available on Windows 10

### DISCUSSION THREAD

**Discussion by jazzfog** [microsoft/wslg#495] [Q&A] [answered]
...and when?

Or just Windows 11?

Thank you.

**Comment by crramirez** [comment, marked as answer]
Just Windows 11. There are other threads that explain this topic better.

**Reply by jazzfog** [reply]
Thank you. Bad news though.

**Reply by jazzfog** [reply]
Looked around quickly but could not find any details, just statements.
Do you happen to have a link to a topic? I wonder if that's a there is a technical reason or just marketing...

**Reply by crramirez** [reply]
Here is the official statement: https://github.com/microsoft/wslg/issues/347#issuecomment-877695975 but no technical details, but in this issue, you can find a lot of information. There is a feature request that you can vote up: https://github.com/microsoft/wslg/issues/414#issuecomment-938086913

**Reply by jazzfog** [reply]
Thank you again @crramirez!
